app/models/printables/group.rb

```diff
@@ -26,11 +27,11 @@ def classify_for_printing(printer_config)
   def print(printer_config, _username = nil)
     return if Rails.configuration.printing_disabled
 
-    classify_for_printing(printer_config).each do |(printer_name, external_id), assets|
+    classify_for_printing(printer_config).each do |(printer_name, label_template), assets|
       body_print = assets.filter_map(&:printable_object).reverse
       next if body_print.empty?
 
-      PMB::PrintJob.new(printer_name: printer_name, label_template_id: external_id, labels: { body: body_print }).save
+      PrintMyBarcodeJob.new(printer_name: printer_name, label_template: label_template, labels: body_print).save
```

Comment:
How come are we still supporting both?

Reply:
Just to let us roll-back quickly if there is a problem - its not quite a feature flag, as it will need a config deployment - but it means we wont need to change the code. Will remove it soon afterwards.